src/load-config.js

```diff
@@ -10,7 +10,10 @@ const projectRoot = path.dirname(path.dirname(fileURLToPath(import.meta.url)));
 debug(`project root: '${projectRoot}'`);
 export const sqlDir = path.join(projectRoot, 'src', 'sql');
 
-let defaultDataDir = fs.existsSync('/data') ? '/data' : projectRoot;
+let defaultDataDir = fs.existsSync(path.join(projectRoot, 'config.json'))
+  ? projectRoot
+  : '/data';
```

Comment:
Ok, this won't really work. So folks can define a custom `data` (database) location. For example: dropbox. And we should always respect the users choice in this.

However `config.json` is different. For this specific file we could implement the logic you are proposing here. But only for `config.json` file. Not for `/data`.

Does that make sense?

Reply:
I see, thanks for bringing that up. I'll fix this shortly.